File: JAVA/Operators/ConditionalStatement/src/Fibonacci.java
public class Fibonacci {
    public static void main(String args[]) {
        int a = 0;
        int b = 1;
        int n = 9;
        System.out.print(a + "," + b);
        for (int i = 0; i < n-2; i++) {
            int c = a + b;
            a = b;
            b = c;
            System.out.print("," + c);
        }
    }
}

File: DSA/Introduction Pograms/fibonacci.cpp
#include <iostream>
using namespace std;
void fibonacci(int n)
{
    int f = -1;
    int s = 1;
    for (int i = 1; i <= n; i++)
    {
        int sum = f + s;
        cout << sum << " ";
        f = s;
        s = sum;
    }
}
int main()
{
    system("cls");
    int n = 8;
    fibonacci(n);
    return 0;
}